server/README.md

```diff
@@ -128,7 +128,7 @@ curl http://localhost:5000/api/user -H "Content-Type: application/json" --data '
 
 Update User:
 ```sh
-curl -X PUT http://localhost:5000/api/user/607ae876f59d21d5a50ac3c9 -H "Content-Type: application/json" --data '{"pseudo":"un autre pseudo"}'
+curl -X PUT http://localhost:5000/api/user/607ae876f59d21d5a50ac3c9 -H "Content-Type: application/json" --data '{"pseudo":"un autre pseudo", "profileImage": "link to profile image"}'
```

Comment:
Tested but doesn't work, there is an error in the field name on the ticket, it's called `profilImage` (on the server) not `profileImage` ([see model](https://github.com/mdl29/scratchy/blob/main/server/scratchy_server/model/userModel.py#L6)).

Can you update it please ? Thanks in advance for your help.

Reply:
Refactored profileImage attribute of UserModel instead.